datafusion/core/tests/fuzz_cases/aggregation_fuzzer/fuzzer.rs

```diff
@@ -305,3 +327,172 @@ impl AggregationFuzzTestTask {
         )
     }
 }
+
+/// Pretty prints the `RecordBatch`es, limited to the first 100 rows
+fn format_batches_with_limit(batches: &[RecordBatch]) -> impl std::fmt::Display {
```

Comment:
without limiting the size of the output, the output is overwhelming

Reply:
I think maybe it is unnecessary to output batches?

Reply:
I found that seeing the differences in the output made it easier for me to understand what was wrong

Specifically, when I added a test for `AVG` and it failed intermittently for me, it was easy for me to say "integer overflow" as the values of the averages were `e19` or something like that